### PR TITLE
Typed all policy uniques

### PR DESCRIPTION
**Pull request by xlenstra**
Turns out there are a lot of uniques in the `policies.json` files.
All of them (except the timed bonus, which I'll handle separately sometime) should be typed now.
Also reworded a few to get rid of + and - signs outside of parameters, as well as turning 'increased by [amount]' into '[+amount]', etc.

### CODE REVIEW

File: android/assets/jsons/Civ V - Gods & Kings/Buildings.json [hunk 1]
```diff
@@ -477,7 +477,7 @@
 		"happiness": 4,
 		"greatPersonPoints": {"Great Engineer": 1},
 		"isWonder": true,
-		"uniques": ["Golden Age length increased by [50]%"],
+		"uniques": ["[+50]% length of Golden Ages"],
```
Comment (yairm210) [inline android/assets/jsons/Civ V - Gods & Kings/Buildings.json:480]: Of? For?

File: android/assets/jsons/Civ V - Gods & Kings/Buildings.json [hunk 2]
```diff
@@ -1008,7 +1008,7 @@
 		"culture": 5,
 		"isWonder": true,
 		"greatPersonPoints": {"Great Artist": 2},
-		"uniques": ["Culture cost of adopting new Policies reduced by [10]%"],
+		"uniques": ["[-10]% Culture cost of adopting new Policies"],
```
Comment (yairm210) [inline android/assets/jsons/Civ V - Gods & Kings/Buildings.json:1011]: Here too, maybe "for adopting" not "of adopting"?

**Review by yairm210 — APPROVED** [review]
This is pretty crazy

File: android/assets/jsons/Civ V - Gods & Kings/Buildings.json (hunk 1 above)
Comment (will-ca) [reply android/assets/jsons/Civ V - Gods & Kings/Buildings.json:480]: As it's being re-worded anyway, maybe "+XX% Golden Age length" or "+XX Golden Age duration"?

File: android/assets/jsons/Civ V - Gods & Kings/Buildings.json (hunk 2 above)
Comment (will-ca) [reply android/assets/jsons/Civ V - Gods & Kings/Buildings.json:1011]: IMO "of" is right.

Eh, NGram viewer:

https://books.google.com/ngrams/graph?content=cost+of%2C+cost+for&year_start=1800&year_end=2019&corpus=26&smoothing=3&direct_url=t1%3B%2Ccost%20of%3B%2Cc0%3B.t1%3B%2Ccost%20for%3B%2Cc0

Comment (xlenstra) [reply android/assets/jsons/Civ V - Gods & Kings/Buildings.json:1011]: As it already used to be 'of' in the old unique, I'm inclined to keep using 'of'

File: core/src/com/unciv/models/ruleset/unique/UniqueType.kt [hunk 3]
```diff
@@ -166,13 +184,32 @@ enum class UniqueType(val text:String, vararg targets: UniqueTarget, val flags:
...
+    BonusHappinessFromLuxury("[amount] Happiness from each type of luxury resource", UniqueTarget.Global),
+    @Deprecated("As of 3.18.17", ReplaceWith("[+amount] Happiness from each type of luxury resource"))
+    BonusHappinessFromLuxuryDeprecated("+[amount] happiness from each type of luxury resource", UniqueTarget.Global),
+    LessPolicyCostFromCities("Each city founded increases culture cost of policies [amount]% less than normal", UniqueTarget.Global),
+    LessPolicyCost("[amount]% Culture cost of adopting new policies", UniqueTarget.Global),
+    @Deprecated("As of 3.18.17", ReplaceWith("[amount]% Culture cost of adopting new policies"))
+    LessPolicyCostDeprecated("Culture cost of adopting new Policies reduced by [amount]%", UniqueTarget.Global),
+
+    // Should the 'R' in 'Research agreements' be capitalized?
     EnablesResearchAgreements("Enables Research agreements", UniqueTarget.Global),
```
Comment (will-ca) [inline core/src/com/unciv/models/ruleset/unique/UniqueType.kt:211]: If it is, then the "A" in "Agreements" should also be capitalized.

Actually, this should probably be standardized across all proper-noun game mechanics… Will open new issue.

**Review by will-ca — APPROVED** [review]

**Comment by yairm210** [comment]
Let's agree to try and capitalize all game terms going forward, especially in uniques, but I don't see this as a major impediment

**Comment by will-ca** [comment]
> Let's agree to try and capitalize all game terms going forward, especially in uniques, but I don't see this as a major impediment

Wait, the different capitalizations of `LessPolicyCost` seem like more than an aesthetic issue. Are uniques case-insensitive?

**Comment by yairm210** [comment]
Uniques are case sensitive

**Pull request merged**